src/main/java/nextstep/courses/service/EnrollmentService.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.courses.service;
+
+import nextstep.courses.domain.PaymentRepository;
+import nextstep.courses.domain.Session;
+import nextstep.courses.domain.SessionRepository;
+import nextstep.payments.domain.Payment;
+import nextstep.qna.NotFoundException;
+import nextstep.users.domain.Enrollment;
+import nextstep.users.domain.NsUser;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class EnrollmentService {
+
+    private final PaymentRepository paymentRepository;
+    private final SessionRepository sessionRepository;
+
+    public EnrollmentService(PaymentRepository paymentRepository, SessionRepository sessionRepository) {
+        this.paymentRepository = paymentRepository;
+        this.sessionRepository = sessionRepository;
+    }
+
+    @Transactional
+    public Enrollment enroll(NsUser nsUser, Long sessionId) {
+        Session session = sessionRepository.findById(sessionId).orElseThrow(() -> new NotFoundException(String.format("%d session을 찾을 수 없습니다.", sessionId)));
```

Comment:
```suggestion
        Session session = sessionRepository.findById(sessionId)
        .orElseThrow(() -> new NotFoundException(String.format("%d session을 찾을 수 없습니다.", sessionId)));
```
개행을 둬서 가독성을 높여보면 어떨까요?

Reply:
넵 수정하겠습니다!